hastory-server/src/Data/Hastory/Server/Utils.hs

```diff
@@ -0,0 +1,33 @@
+{-# LANGUAGE RecordWildCards #-}
+
+module Data.Hastory.Server.Utils where
+
+import Crypto.Hash (Digest, SHA256(..), hashWith)
+import qualified Data.ByteString.Char8 as C
+import Data.Hastory.Types (Entry(..), ServerEntry(..), SyncRequest(..))
+import qualified Data.Text as T
+import qualified Data.Text.Encoding as T
+import Network.HostName (HostName)
+
+hashEntry :: Entry -> HostName -> Digest SHA256
```

Comment:
I don't think we want to use a `String` as this parameter. Instead I think we should stick with `Text` because you use `T.unpack` later on in this file and that is then not necessary anymore.

Reply:
Of course! 👍